common/constant/generic.go

```diff
@@ -0,0 +1,6 @@
+package constant
+
+const (
+	GENERIC_SERIALIZATION_DEFAULT = "true"
```

Comment:
`GENERIC_SERIALIZATION_DEFAULT` should be set to `hessian` instead of `true`.

Reply:
The use of `true` is for backward compatibility and consistency with dubbo.

Reply:
OK

Reply:
You are right, but I still think here should support `hessian` field to help the people who are new to dubbo-go, in order to make compatible for the older system, `true` could be supported and represents hessian at the same time.